### ISSUE TITLE
formatting is no longer working

### ISSUE THREAD

**Issue by shrkw** [r7kamura/vscode-ruby-light#32]
It seems that the formatting function of vscode-ruby-light is no longer working. It appears to be an error related to web-tree-sitter. When I upgraded from version 0.20.5 to 0.20.8, it worked as expected.

```
[Info  - 12:03:07] Connection to server got closed. Server will restart.
node:internal/deps/undici/undici:11413
    Error.captureStackTrace(err, this);
          ^

TypeError: Failed to parse URL from /Users/vscode/.vscode/extensions/r7kamura.vscode-ruby-light-0.5.2/server/node_modules/web-tree-sitter/tree-sitter.wasm
    at Object.fetch (node:internal/deps/undici/undici:11413:11) {
  [cause]: TypeError [ERR_INVALID_URL]: Invalid URL
      at new NodeError (node:internal/errors:399:5)
      at URL.onParseError (node:internal/url:566:9)
      at new URL (node:internal/url:646:5)
      at new Request (node:internal/deps/undici/undici:7002:25)
      at fetch2 (node:internal/deps/undici/undici:10554:25)
      at Object.fetch (node:internal/deps/undici/undici:11411:18)
      at fetch (node:internal/process/pre_execution:248:25)
      at /Users/vscode/.vscode/extensions/r7kamura.vscode-ruby-light-0.5.2/server/node_modules/web-tree-sitter/tree-sitter.js:1:15192
      at /Users/vscode/.vscode/extensions/r7kamura.vscode-ruby-light-0.5.2/server/node_modules/web-tree-sitter/tree-sitter.js:1:15413
      at new Promise (<anonymous>) {
    input: '/Users/vscode/.vscode/extensions/r7kamura.vscode-ruby-light-0.5.2/server/node_modules/web-tree-sitter/tree-sitter.wasm',
    code: 'ERR_INVALID_URL'
  }
}

Node.js v18.15.0
[Error - 12:03:07] Connection to server got closed. Server will not be restarted.
```

**Comment by r7kamura** [comment]
Thanks for reporting @shrkw!
There is no strong reason to fix web-tree-sitter version to 0.20.5, so we should change it to 0.20.8 or later.
I appologize, but I am not actively using this extension at the moment, so I may be a little late in responding to this issue. If you make sure it works and send me a pull request, I will promptly approve it and release new version.

**Comment by shrkw** [comment]
Exactly, I moved to shopify.ruby-extensions-pack, and works fine. I'll close this.
Lastly, thank you for creating and publishing such a nice product. It has been incredibly helpful thanks to you.
